src/app/descriptors/budget/budget8dig/text-template.html

```diff
@@ -36,4 +36,11 @@
 {{contractors_allocated}} חודשי עבודה בלתי צמיתה (יועצים וקבלנים)
 {% endif %}.
 <br/>
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+{% if year > charts[0].subcharts[0].chart[0].x[0] %}
```

Comment:
This is much better - but please take this information from the item's history and not assume the type of the first chart.